src/chunkserver/network_worker_thread.cc

```diff
@@ -1215,36 +1267,35 @@ void worker_fwdread(csserventry *eptr) {
 				return;
 			}
 			stats_bytesin += i;
-			eptr->fwdinputpacket.startptr += i;
-			eptr->fwdinputpacket.bytesleft -= i;
-			if (eptr->fwdinputpacket.bytesleft > 0) {
+			eptr->fwdInputPacket.startPtr += i;
+			eptr->fwdInputPacket.bytesLeft -= i;
+			if (eptr->fwdInputPacket.bytesLeft > 0) {
 				return;
 			}
 		}
-		ptr = eptr->fwdhdrbuff;
+		ptr = eptr->fwdHeaderBuffer;
 		type = get32bit(&ptr);
 		size = get32bit(&ptr);
 
-		eptr->fwdmode = HEADER;
-		eptr->fwdinputpacket.bytesleft = 8;
-		eptr->fwdinputpacket.startptr = eptr->fwdhdrbuff;
+		eptr->fwdMode = ChunkserverEntry::Mode::Header;
+		eptr->fwdInputPacket.bytesLeft = PacketHeader::kSize;
+		eptr->fwdInputPacket.startPtr = eptr->fwdHeaderBuffer;
 
-		worker_gotpacket(eptr, type, eptr->fwdinputpacket.packet, size);
+		worker_gotpacket(eptr, type, eptr->fwdInputPacket.packet, size);
 
-		if (eptr->fwdinputpacket.packet) {
-			free(eptr->fwdinputpacket.packet);
+		if (eptr->fwdInputPacket.packet) {
+			free(eptr->fwdInputPacket.packet);
 		}
-		eptr->fwdinputpacket.packet = NULL;
+		eptr->fwdInputPacket.packet = nullptr;
 	}
 }
 
-void worker_fwdwrite(csserventry *eptr) {
+void worker_fwdwrite(ChunkserverEntry *eptr) {
 	TRACETHIS();
 	int32_t i;
-	if (eptr->fwdbytesleft > 0) {
-		i = write(eptr->fwdsock, eptr->fwdstartptr, eptr->fwdbytesleft);
+	if (eptr->fwdBytesLeft > 0) {
+		i = write(eptr->fwdSocket, eptr->fwdStartPtr, eptr->fwdBytesLeft);
```

Comment:
Same suggestion here for the name of variable `i`.

Reply:
Applied here: https://github.com/leil-io/saunafs/commit/8f8dcf733ccd489fa6b9b4426b1ec5629500c839
Please notice this is out of the scope of this change.